### ISSUE TITLE
Fill in PHP field registration in README

### ISSUE THREAD

**Issue by wdews** [wp-graphql/wp-graphql-acf#186]
In the readme.md file, the section titled [Registering Fields in PHP](https://github.com/wp-graphql/wp-graphql-acf#registering-fields-in-php) trails off with a `@todo` annotation.

Could somebody please fill in this section?

**Comment by khaosophy** [comment]
I second this. I'm trying to figure out how add PHP-produced ACF fields to the GraphQL. If anyone can point me in the right direction, I'd be happy to write up the README once I figure it out.

**Comment by arnespremberg** [comment]
In the issue I link below it's mentioned that this is currently under development. However it appears you need to add an entry `'show_in_graphql'` to the field group array and set it to `1`. additionally it seems you could add a `'graphql_field_name'` value.

However I haven't tested that yet


https://github.com/wp-graphql/wp-graphql-acf/issues/188

**Comment by arnespremberg** [comment]
Please see the following code snippet that worked for me.

```php
acf_add_local_field_group(array(
	'key' => 'test_group',
	'title' => 'test_group',
	'fields' => array(
		array(
			'key' => 'test_field,
			'label' => 'test_field',
			'name' => 'test_field',
			'type' => 'text',
			'show_in_graphql' => 1,
		),
	),
	'location' => array(
		array(
			array(
				'param' => 'post_type',
				'operator' => '==',
				'value' => 'post',
			),
		),
	),
	'show_in_graphql' => 1,
	'graphql_field_name' => 'testfield',
));
```

**Comment by wdews** [comment]
@arnespremberg the concern was not that I couldn't get this approach working, but that this was not documented in the README to prevent having to essentially reverse engineer the plugin in order to understand how to surface fields correctly.

**Comment by arnespremberg** [comment]
@wdews
If I find the time and the maintainers are happy with it I'll make a PR with a small paragraph about it. Maybe you got the time to do it right away?

Oh, and by the way, there was another person in this issue that seemed to struggle with finding the right option.

**Comment by khaosophy** [comment]
I'm gonna fiddle with it this afternoon (or potentially into tomorrow) to confirm it works and I'll put in a PR with the update. It occurs to me my issue was more along the lines of custom post types, but those CPTs did have custom fields. Honestly they seemed to work pretty much out of the box, but I'm setting up an instance from scratch to confirm.

**Comment by jasonbahl** [comment]
Closed by #235
